code.js

```diff
@@ -2,44 +2,36 @@
 //Maxie Machado 
 // Dijkstra's Algorithm 
 
-function dijkstra(graph, sourceNode) 
-{
+function dijkstra(graph, sourceNode) {
     let dist = {};
-    let vis = new Set()
+    let vis = new Set();
 
-    for (let node in graph)
-    {
+    for (let node in graph) {
         dist[node] = Infinity;
     }
     dist[sourceNode] = 0;
 
-    function getClosestUnvisitedNode()
-    {
+    function getClosestUnvisitedNode() {
         let minDist = Infinity;
         let clstNode = null;
 
-        for (let node in dist)
-        {
-            If (!vis.has(node) && dist[node] < minDist)
-            {
+        for (let node in dist) {
+            if (!vis.has(node) && dist[node] < minDist) {
                 minDist = dist[node];
-                clstNode = node; 
+                clstNode = node;
             }
         }
         return clstNode;
     }
 
     let currentNode = getClosestUnvisitedNode();
 
-    while (currentNode !== null)
-    {
+    while (currentNode !== null) {
         let currDist = dist[currentNode];
         vis.add(currentNode);
 
-        for (let neighbor in graph[currentNode])
-        {
-            if (!vis.has(neighbor))
-            {
+        for (let neighbor in graph[currentNode]) {
+            if (!vis.has(neighbor)) {
                 let newDist = currDist + graph[currentNode][neighbor];
                 dist[neighbor] = Math.min(dist[neighbor], newDist);
             }
```

Comment:
Thanks. One more thing. You need to create a YAML file to test your test code automatically. Your Actions are empty right now. In your project’s root directory, create a .github/workflows directory. Then, create a file named test.yml (you can name it whatever you like). Inside the YAML file should look like this: 

on:
  push:

jobs:
  test:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v3
    - run: npm i jsverify
    - run: node code.test.js